### PR TITLE
added live demo link in README file

### PR DESCRIPTION
**Pull request by pritampatil4**

### CODE REVIEW

**Comment by hjhill** [comment]
@pritampatil4 how can adding something to the README change `package.lock`?

File: README.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ and to disable the hard redirect, and do it using javascript routing.
 
 Open <http://localhost:3000/>
 
+Live Demo - https://optile.github.io/demo-express-checkout-react-spa
```
Comment (z0r0132) [inline README.md]:
```suggestion
Live Demo - <https://optile.github.io/demo-express-checkout-react-spa>
```

File: package-lock.json [hunk 2]
```diff
@@ -1685,7 +1685,7 @@
       "dependencies": {
         "core-js": {
           "version": "2.6.11",
-          "resolved": "https://registry.npmjs.org/core-js/-/core-js-2.6.11.tgz",
```
Comment (z0r0132) [inline package-lock.json]: The users of this repository should get the dependencies from npmjs and not from nexus, please update this file to link to npmjs, sam efor all other dependencies in this file

Comment (pritampatil4) [reply package-lock.json]: Done

**Pull request merged**